packages/loopring_v3/contracts/impl/libexchange/ExchangeBalances.sol

```diff
@@ -144,15 +145,15 @@ library ExchangeBalances
         uint     pubKeyY,
         uint     nonce,
         uint     balancesRoot,
-        uint[30] memory accountMerkleProof
+        uint[36] memory accountMerkleProof
         )
         private
         pure
         returns (uint)
     {
         uint accountItem = hashImpl(pubKeyX, pubKeyY, nonce, balancesRoot);
         uint _id = accountID;
-        for (uint depth = 0; depth < 10; depth++) {
+        for (uint depth = 0; depth < 12; depth++) {
```

Comment:
We are only changing the account tree depths, right, so why change 10 to 12?

Reply:
This is for the accounts tree. The Merkle root of the accounts tree is stored onchain so we need to get to that value. The balance Merkle trees are children of the accounts so we need to pass to go through the balance Merkle tree for the account and then through the accounts Merkle tree to get to the Merkle tree root stored onchain.

Reply:
I see:
10 == 20/2 (2^20 accounts)
12 == 24/2（2^24 accounts)